src/Tizen.NUI/src/public/Common/PropertyValue.cs

```diff
@@ -244,6 +244,39 @@ internal PropertyValue(AngleAxis angleAxis) : this(Interop.PropertyValue.NewProp
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Determines whether the ProperyValue has equal value with the current ProperyValue.
+        /// </summary>
+        /// <remarks>
+        /// Equal only if same type.
```

Comment:
hope we note more about why we need to use this function not Equals().

Reply:
There is some sad story. Some objects (like Color & Vector4) already used Equals(), without valid GetHashCode().
Before explain the reason of `not Equals() API`, we should fix them.